### ISSUE TITLE
what database?

### ISSUE THREAD

**Issue by joepareti54** [MIT-LCP/Shakespeare-Method#3]
before executing the notebook 1.0_create_adult_inputs.ipynb I have created a database as follows:
`
make mimic  datadir="/media/joepareti54/Elements/x/finance-2021/AI/NLP/Shakespeare-Booz-Allen/physionet.org/files/mimiciii-demo/1.4/" DBNAME="joepareti54" DBPASS="***” `

and then changed the notebook:
`from sqlalchemy import create_engine, update
conn = psycopg2.connect("dbname=joepareti54 user=postgres \
                         password=**** options=--search_path=mimiciii");
engine = create_engine('postgresql://joepareti54:****@localhost/mimiciii')
cur = conn.cursor();
cur.execute("""SET search_path = mimiciii;""")`

because I do not have the variables 
`POSTGRES_CONNECT = os.environ.get("POSTGRES_CONNECT")
POSTGRES_ENGINE = os.environ.get("POSTGRES_ENGINE")`

The notebook runs although the data is different than [here](https://github.com/MIT-LCP/Shakespeare-Method/tree/main/pdf)

The notebook 1.1a_create_transfused_cohort.ipynb however throws an invalid syntax exception, and I believe it does not find any transfusion case:

`COPY mimiciii.transfusion_items_dict 
FROM 'D:\\20180717D_ITEMS_related_to_blood_full.csv' 
DELIMITER ',' CSV HEADER;`

In summary, I need to identify the correct data-set and to fix the above code. More information [here](https://docs.google.com/document/d/1a-kd3EskL3EhhsWPnZSOHslRRWFpdmpxhu491aPSOfE/edit?usp=sharing).
